tequila_fab/__init__.py

```diff
@@ -0,0 +1,68 @@
+"""
+All tasks should have Usage: lines in their docstrings.
+See existing tasks for examples.  This allows users
+to see an overview of tasks by just running
+``fab --list``.
+
+Add tasks that should be exported to ``__ALL__``.
+"""
+
+from fabric.api import env, local, require, task
+from fabric.tasks import execute
+
+from .ansible import check_role_versions, install_roles
+
+#
+# KEEP TASKS IN ALPHABETICAL ORDER
+#
+__ALL__ = [
+    'bootstrap',
+    'check_role_versions',
+    'create_superuser',
+    'deploy',
+    'install_roles',
+]
+
+
+#
+# KEEP TASKS IN ALPHABETICAL ORDER
+#
+@task
+def bootstrap():
+    """
+    Usage: fab <ENV> bootstrap
+    """
+    install_roles()
+    execute(check_role_versions)
+    deploy("bootstrap_python")
+    deploy("site", extra_vars='{"unmanaged_users": [ubuntu]}')
+
+
+@task
+def create_superuser(email):
+    """
+    Usage: fab <ENV> create_superuser:dpoirier@caktusgroup.com
+    """
+    require('environment')
+    deploy('create_superuser', extra_vars={'EMAIL': email})
```

Comment:
I don't have this playbook in my project and is likely something others will encounter as we add more tasks here. Maybe we could start to also keep [caktus/tequila](https://github.com/caktus/tequila/tree/master/playbooks) updated with current playbooks.